### PR TITLE
feat(pdp): display no of items left

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**

### CODE REVIEW

File: components/KiboProductActions.vue [hunk 1]
```diff
@@ -10,7 +10,9 @@
           class="sf-add-to-cart__select-quantity quantity-input"
           @input="$emit('input', $event)"
         />
-        <div class="quantity-left">{{ quantityLeft }} item(s) left</div>
+        <div v-show="isValidForAddToCart" class="quantity-left">
+          {{ quantityLeft }} item(s) left
```
Comment (kibo-chandradeeptalaha) [inline components/KiboProductActions.vue]: create localization for the whole string. use pluralization as well.

File: lib/getters/productGetters.ts [hunk 2]
```diff
@@ -161,6 +161,26 @@ const getSegregatedOptions = (product: ProductCustom) => {
 const validateAddToCart = (product: ProductCustom): boolean =>
   isProductVariationsSelected(product) && Boolean(product.fulfillmentMethod)
 
+const getItemsLeft = (
```
Comment (kibo-chandradeeptalaha) [inline lib/getters/productGetters.ts]: can we change the name to getAvailableItemCount ?

File: lib/getters/productGetters.ts [hunk 3]
```diff
@@ -161,6 +161,26 @@ const getSegregatedOptions = (product: ProductCustom) => {
...
+  product: ProductCustom,
+  productLocationInventoryData: Object,
+  fulfillmentOptionValue: string
+): number => {
+  const allVariantSelected = isProductVariationsSelected(product)
+  const qtyLeft = { value: 0 }
```
Comment (kibo-chandradeeptalaha) [inline lib/getters/productGetters.ts:170]: we can make it a number datatype as well !

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please take a look

File: components/KiboProductActions.vue (hunk 1 above)
Comment (kibo-susantasamanta) [reply components/KiboProductActions.vue]: Done

File: lib/getters/productGetters.ts (hunk 2 above)
Comment (kibo-susantasamanta) [reply lib/getters/productGetters.ts]: Yes we can. Changed.

File: lib/getters/productGetters.ts (hunk 3 above)
Comment (kibo-susantasamanta) [reply lib/getters/productGetters.ts:170]: Type added.

File: composables/useProduct.ts [hunk 4]
```diff
@@ -26,6 +26,20 @@ export const useProduct = (referenceKey: string) => {
     }
     loading.value = false
   }
+  const getProductLocationInventory = async (productCode: string, locationCodes: string) => {
```
Comment (kibo-kevinwatts) [inline composables/useProduct.ts]: this should be its own composable

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: composables/useProduct.ts (hunk 4 above)
Comment (kibo-susantasamanta) [reply composables/useProduct.ts]: Composable for ProductLocationInventory is created.

**Review by kibo-kevinwatts — APPROVED** [review]

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by kibo-sushant — APPROVED** [review]

**Pull request merged**